mythril/analysis/modules/dependence_on_predictable_vars.py

```diff
@@ -54,171 +48,41 @@ def _analyze_states(state: GlobalState) -> list:
     :return:
     """
     issues = []
-    call = get_call_from_state(state)
-    if call is None:
-        return []
-    if "callvalue" in str(call.value):
-        log.debug("[DEPENDENCE_ON_PREDICTABLE_VARS] Skipping refund function")
-        return []
 
-    # We're only interested in calls that send Ether
-    if call.value.type == VarType.CONCRETE and call.value.val == 0:
-        return []
-
-    address = call.state.get_current_instruction()["address"]
-
-    description = (
-        "The contract sends Ether depending on the values of the following variables:\n"
-    )
-
-    # First check: look for predictable state variables in state & call recipient constraints
+    # Look for predictable state variables in jump condition
 
     vars = ["coinbase", "gaslimit", "timestamp", "number"]
```

Comment:
What do you think about using taint vs the variable name to check if the variable is created by one of these?

Reply:
Yes, that's much cleaner.